src/components/custom/_signup-new.tsx

```diff
@@ -116,42 +116,23 @@ const SocialButton = styled(Button)<SocialButtonProps>`
     display: inline-flex;
     justify-content: center;
     align-items: center;
+    border: none;
     box-shadow: none;
     background-color: ${(props) => props.bgColor || 'var(--color-white)'};
-    border: solid 1px var(--color-grey-21);
-    width: 12.5rem;
     height: 3.8rem;
-    padding: 0.5rem 0;
-
-    &:hover {
```

Comment:
Don't we need this `hover` color change? Did we check this with design team?

Reply:
Its not required. Since, last time the button has white background. But, now its only svg icons with their default background color. The implementation of the button same as in `_signup-public.tsx`